walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -47,7 +47,8 @@
       if (!Collections.disjoint(event.getAttendees(), attendees)) {
         TimeRange eventTime = event.getWhen();
         
-        for (int i = 0; i < possibleMeetingTimes.size(); i++) {
+        int i = 0;
```

Comment:
nit: when fixing bugs, can you write a corresponding test to prevent this from happening in the future?